### ISSUE TITLE
npm package 1.4.5 appears to be bad

### ISSUE THREAD

**Issue by guillaumevincent** [fraywing/textAngular#888]
Hi,

`npm install textangular` appears to download a strange release.

"textangular": "^1.4.5"

Content of node_modules/textangular/dist/textAngular.css

```
/*
@license textAngular
Author : Austin Anderson
License : 2013 MIT
Version 1.4.5

See README.md or https://github.com/fraywing/textAngular/wiki for requirements and use.
*/

/*
Commonjs package manager support (eg componentjs).
*/

/* istanbul ignore next:  */
'undefined'!=typeof module&&'undefined'!=typeof exports&&module.exports===exports&&(module.exports='textAngular');

(function(){ // encapsulate all variables so they don't become global vars
"use strict";.ta-hidden-input {
	width: 1px;

```

I did a `npm cache clean`

**Comment by pbassut** [comment]
This was removed on 1.4.5. There must be some cache problem with your version.

**Comment by guillaumevincent** [comment]
@pbassut I tested on a new computer I got the same, the package is not good.
I clean my cache.

my coworkers got the same error.

```
  "dist": {
    "shasum": "90eacbb311440d05b5482d558170e7eb064fc686",
    "tarball": "http://registry.npmjs.org/textangular/-/textangular-1.4.5.tgz"
  },
  "directories": {},
  "_resolved": "https://registry.npmjs.org/textangular/-/textangular-1.4.5.tgz",
  "readme": "ERROR: No README data found!"
```
the tgz on the main registry is not good (http://registry.npmjs.org/textangular/-/textangular-1.4.5.tgz)

**Comment by pbassut** [comment]
Could you try using bower? I guess there was a problem when updating npm.

**Comment by guillaumevincent** [comment]
@pbassut Bower use tar.gz package from github, so I confirm Bower package is good. I just want to let you know that npm package 1.4.5 is not good :)

I already use bower

**Comment by pbassut** [comment]
Thanks for the report. I think we had an issue where the version 1.4.5 was already publish. Then I'm not sure if we published 1.4.6 or waited for the next release or something. I didn't quite get it what happened. 

Thanks for letting us know.

**Comment by JoelParke** [comment]
updated in v1.4.6
